examples/ping4d/suppressions.txt

```diff
@@ -0,0 +1,2 @@
+# This is a known leak.
```

Comment:
maybe add a little more comment here to say this file is used for sanitize checks on gha build.